pkg/cli/idle/idle.go

```diff
@@ -643,6 +643,11 @@ func (o *IdleOptions) RunIdle() error {
 			hadError = true
 			continue
 		}
+		// this CLI command is "do the thing" and doesn't provide a sensible input object for a starting ResourceVersion
+		// the command is an imperative, "do this now" without a precondition, so the desired behavior is deconflicting.
+		// we can do this by having an unconditional scale instance for writing.
+		scale.ResourceVersion = ""
```

Comment:
Won't this potentially end up reverting actual important changes that someone else had made to the object?

Reply:
The scale subresource is limited to only the replicas field, so it will only do the scale ot zero.

Reply:
Reference here https://github.com/kubernetes/kubernetes/blob/2db1b321e0edf9e3c4e434353d505cec96bfb319/test/e2e/apps/deployment.go#L1669